README.md

```diff
@@ -17,6 +17,7 @@ A curated list of amazingly awesome awesomeness.
 	- Common Lisp
 		- [Common Lisp Libraries](https://github.com/CodyReichert/awesome-cl)
 		- [Learning Common Lisp](https://github.com/GustavBertram/awesome-common-lisp-learning-list)
+		- [Coronavirus](https://github.com/soroushchehresa/awesome-coronavirus)
```

Comment:
Can you fix the indent? Currently your list is displayed under the Common Lisp awesome lists.